example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -278,33 +279,43 @@ describe('The S3 Ingest Granules workflow', () => {
     ]);
   });
 
+  beforeEach(() => {
+    if (beforeAllError) fail(beforeAllError);
+  });
+
   it('prepares the test suite successfully', () => {
-    if (beforeAllFailed) fail('beforeAll() failed to prepare test suite');
+    if (beforeAllError) fail(`beforeAll() failed to prepare test suite: ${testSetupError}`);
```

Comment:
I think `testSetupError` is undefined

Reply:
Right you are.   Also because of the beforeEach, largely unneeded: 70d163df0